flink-runtime/src/test/java/org/apache/flink/runtime/io/network/netty/NettyMessageSerializationTest.java

```diff
@@ -174,6 +197,18 @@ private void testEncodeDecodeBuffer(boolean testReadOnlyBuffer) {
 		assertEquals(expected.backlog, actual.backlog);
 	}
 
+	private Buffer compress(Buffer buffer) {
```

Comment:
This separate method is actually not needed, because only one usage and only one sentence in it.